hw/top_earlgrey/lint/top_earlgrey_dv_lint_cfgs.hjson

```diff
@@ -51,6 +51,32 @@
                   import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
                   rel_path: "hw/ip/i2c/dv/lint/{tool}"
              },
+             {    name: keymgr
+                  fusesoc_core: lowrisc:dv:keymgr_sim
+                  import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
+                  rel_path: "hw/ip/keymgr/dv/lint/{tool}"
+             },
+             {    name: otp_ctrl
+                  fusesoc_core: lowrisc:dv:otp_ctrl_sim
+                  import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
+                  rel_path: "hw/ip/otp_ctrl/dv/lint/{tool}"
+             },
+             // TODO: enable prim_lfsr lint check
+             // {    name: prim_lfsr
```

Comment:
Hi Michael @msfschaffner , this `prim_lsfr` lint gives me warnings for `Non-type localparam names must be styled with CamelCase [Style: constants] [parameter-name-style]` in [top_pkg.sv](https://github.com/lowRISC/opentitan/blob/master/hw/top_earlgrey/rtl/top_pkg.sv#L8). Is there a waiver for this? Otherwise we might need to change lots of files within DV.

Reply:
Another solution might work is to change it from `localparam` to `parameter`

Reply:
In general, if there are lint errors, we should fix the code. I am however a bit surprised that this causes issues... isn't the `top_pkg.sv` already included and linted in many other lint targets (like top-level lint)? Also, our [style lint policy](https://github.com/lowRISC/opentitan/blob/b8a8591eb205e2082731c85a627bcd23c97373e5/hw/lint/tools/veriblelint/rules.vbl#L16) should allow both `parameters` and `localparams` to be styled ALL_CAPS as well...

Reply:
ah I know what the issue is. we need to amend the core files for the LFSR such that the correct lint policy files are passed to the tools.

this requires the following modifications:

1) we have to add the lint configuration to the `prim_lfsr.core` (design side). you can just copy over the missing parts from another core like `aes.core`:
https://github.com/lowRISC/opentitan/blob/8be9186d4579d60fb17188de66c3ff69aad2d53d/hw/ip/aes/aes.core#L39-L103

2) then we should add a lint target to the `prim_lfsr_sim.core`. you can just copy that from another dv core, e.g. `aes_sim.core`: 
https://github.com/lowRISC/opentitan/blob/8be9186d4579d60fb17188de66c3ff69aad2d53d/hw/ip/aes/dv/aes_sim.core#L28-L39

this should hopefully fix it.

Reply:
Thanks Michael, I see, it is the core file. I was surprised that no other lint caught this warning.